provider/gce/config.go

```diff
@@ -79,3 +80,8 @@ func newConfig(cfg, old *config.Config) (*environConfig, error) {
 	}
 	return ecfg, nil
 }
+
+func (c *environConfig) baseImagePath() (path string, isSet bool) {
```

Comment:
don't need named return values, just (string, bool)

Reply:
Prefer to have the variables in there, so that the method is self-documenting. Even though we're using the value as-is, not using `ok` for its original purpose.